Gems/SceneProcessing/Code/Source/Config/Components/SceneProcessingConfigSystemComponent.cpp

```diff
@@ -34,7 +34,7 @@ namespace AZ
             ActivateSceneModule(SceneProcessing::s_sceneDataModule);
             ActivateSceneModule(SceneProcessing::s_sceneBuilderModule);
             
-            // Defaults in case there's no config setup in the Project Configurator.
+            // Defaults in case there's no config setup in the Project Manager.
```

Comment:
I don't believe Project Manager sets this up but I'm not sure what config it is referring to

Reply:
Will remove this mention